docs/security.html

```diff
@@ -1438,7 +1438,7 @@ <h4><a id="security_authz_examples" href="#security_authz_examples">Examples</a>
             <pre class="brush: bash;">bin/kafka-acls.sh --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:Bob --allow-principal User:Alice --allow-host 198.51.100.0 --allow-host 198.51.100.1 --operation Read --operation Write --topic Test-topic</pre>
             By default, all principals that don't have an explicit acl that allows access for an operation to a resource are denied. In rare cases where an allow acl is defined that allows access to all but some principal we will have to use the --deny-principal and --deny-host option. For example, if we want to allow all users to Read from Test-topic but only deny User:BadBob from IP 198.51.100.3 we can do so using following commands:
             <pre class="brush: bash;">bin/kafka-acls.sh --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:* --allow-host * --deny-principal User:BadBob --deny-host 198.51.100.3 --operation Read --topic Test-topic</pre>
-            Note that ``--allow-host`` and ``deny-host`` only support IP addresses (hostnames are not supported).
+            Note that <code>--allow-host</code> and <code>--deny-host</code> only support IP addresses (hostnames are not supported).
```

Comment:
There's no format like ` `` ` in the documentation anywhere else. 
before:
![image](https://user-images.githubusercontent.com/43372967/82002937-b9367680-9691-11ea-9588-be3628f9c340.png)
after:
![image](https://user-images.githubusercontent.com/43372967/82002970-ce130a00-9691-11ea-8ffc-8ed41b3a55a4.png)